docs-site/docs/autograd.md

```diff
@@ -148,3 +148,978 @@ for epoch in range(epochs):
         for p, gp in autograd.backward(loss):  # auto backward
             sgd.update(p, gp)
 ```
+
+### Supported operators
+
+#### mean
```

Comment:
Copy the comments as the docString for each operator in the python file?
Then we can generate the html for the APIs automatically using Sphinx.

Reply:
has fixed at https://github.com/apache/singa/pull/654